product_variant_default_code/README.rst

```diff
@@ -0,0 +1,167 @@
+============================
+Product Variant Default Code
+============================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fproduct--variant-lightgray.png?logo=github
+    :target: https://github.com/OCA/product-variant/tree/15.0/product_variant_default_code
+    :alt: OCA/product-variant
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/product-variant-15-0/product-variant-15-0-product_variant_default_code
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/webui/builds.html?repo=OCA/product-variant&target_branch=15.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module automatically generate Product Reference (default_code)
+according to attributes data with a configurable behavior.
+
+It defines a reference mask on the product templates so the
+variants references are automatically set. For example:
+
+- Product template: Jacket
+- Attributes:
+  - Color: White, Black
+  - Size: M, L
+- Reference mask: `JKT01-[Color]-[Size]`
+
+- Reference on variants:
+
+  - `JKT01-Wh-M` Jacket White M
+  - `JKT01-Bl-M` Jacket Black M
+  - `JKT01-Wh-L` Jacket White L
+  - `JKT01-Bl-L` Jacket Black L
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
```

Comment:
I'm missing a configuration section with the permissions needed and global configuration options.

Reply:
Please check if the configuration file is correct to regenerate the readme.

Reply:
It looks good to me.